12_glob_lab/globlab.txt

```diff
@@ -3,27 +3,36 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
 1. Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+  
+1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+  echo [a-z]*
+2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+  LC_ALL=C bash -c 'echo [A-Z]*'
+3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
-  4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
+  echo *[aeiou]
+4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
 
-  5. All files with extension ".c" that are in a folder named "src"
+  echo *[!aeiou]
+5. All files with extension ".c" that are in a folder named "src"
   Answer:
 
-  6. All files with extension ".c" under any folder
-  Answer:
+ ls *.c src/ 

```

Comment:
The command you typed shows all *.c files AND files inside src. You were asked to combine so this is a better solution: `ls src/*.c`